pyproject.toml

```diff
@@ -120,6 +123,12 @@ lrs = [
     "python-jose==3.3.0",
     "uvicorn[standard]==0.24.0.post1",
 ]
+full = [
+    "ralph-malph[backends,cli,lrs]",
+]
+all = [
+    "ralph-malph[ci,dev,full]",
```

Comment:
We do not need `ci` in `all` (cf.what you have replaced in the `circleci/config.yam`l)

Reply:
`ci` extra is used only in one place, in the CI job that pushes `ralph-malph` package to PyPi. It would lengthen CI time, so I would keep this extra